ceph_medic/checks/mons.py

```diff
@@ -122,3 +122,19 @@ def check_mon_collocated_with_osd(host, data):
     osd_dirs = get_osd_dirs(dirs)
     if len(osd_dirs):
         return code, msg % ','.join(osd_dirs)
+
+
+def check_mon_recommended_count(host, data):
+    code = 'WMON3'
+    msg = 'Recommended number of MONs (3) not met: %s'
+    mon_count = len(metadata['mons'].keys())
+    if mon_count < 3:
+        return code, msg % mon_count
+
+
+def check_mon_count_is_odd(host, data):
+    code = 'WMON4'
+    msg = 'Recommended number of MONs is not an odd number: %s'
```

Comment:
This reads strangely to me because the recommended number of MONs **is** an odd number and this warning fires when you have an even number of MONs.

Maybe just drop the word "Recommended"?